src/components/pages/doc/admonition/admonition.jsx

```diff
@@ -0,0 +1,47 @@
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const titleClassNames = {
+  note: 'text-[#4C97FF]',
+  important: 'text-[#FFBB33]',
+  tip: 'text-primary-2',
+  warning: 'text-secondary-1',
+  info: 'text-gray-5',
+};
+
+const borderClassNames = {
+  note: 'border-[#4C97FF]',
+  important: 'border-[#FFBB33]',
+  tip: 'border-primary-2',
+  warning: 'border-secondary-1',
+  info: 'border-gray-5',
+};
+
+const Admonition = ({ children, type, title }) => (
```

Comment:
\<nerd-mode> Why not `Warning` or `Note`? \</nerd-mode>

Reply:
Because there are several types of theme as `warning`, `note`, `important`, `info`, `tip`.  It makes sense to use a generic name.